src/ExportPro.IntegrationTests/ExportPro.StorageService.IntegrationTests/Features/ClientFeatures/CreateClient.feature

```diff
@@ -0,0 +1,12 @@
+@CreateClient
+Feature: Creating a client
+
+    Scenario: The user creates a client
+        Given The user is logged in with email  and password  and has necessary permissions
```

Comment:
Given Bob Admin authorized 
When
Then
Bob Admin => json config we need to read crediantials